app/views/pages/map.ejs

```diff
@@ -80,9 +80,26 @@
             a {
                 color: #fff;
             }
+
+            .fas.fa-bars {
+                color: #fff;
+            }
+
+            .hamburger-menu.active {
+                position: relative;
+                display: flex;
+                justify-content: right;
```

Comment:
Hva medfører dette? :)

Reply:
relative for å få den oppå mapet.
Og så skyve den til høyre. Man kunne evt. dytte info boksen til høyre istedet, men de klasjet i hvertfall.

Reply:
http://3488cda4.ngrok.io